contracts/utils/SpigotedLineLib.sol

```diff
@@ -109,7 +109,7 @@ library SpigotedLineLib {
     ) public returns (bool) {
         if (sellToken == Denominations.ETH) {
             // if claiming/trading eth send as msg.value to dex
-            (bool success, ) = swapTarget.call{value: amount}(zeroExTradeData);
+            (bool success, ) = swapTarget.call{value: amount}(zeroExTradeData); // TODO: have we tested this specifically?
```

Comment:
has this been tested with 0x on a mainnet fork with real trade data?

Reply:
nay